applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/settings.js

```diff
@@ -0,0 +1,217 @@
+//'use strict';
+'require view';
+'require form';
+
+return view.extend({
+  render: function() {
+    var m, s, o;
+
+    m = new form.Map('usteer', _('usteer'), _('usteer Configuration.'));
+
+    s = m.section(form.TypedSection, 'usteer', _('options'), _('First four are mandatory'));
+    s.anonymous = true;
+
+    s.option(form.Value, 'network', _('network'), _('The network interface for inter-AP communication)'));
+
+
+
+//    s = m.section(form.TypedSection, 'usteer', _('second section'));
+//    s.anonymous = true;
+
+    o = s.option(form.Flag, 'syslog', _('Syslog'), _('Log messages to syslog (0/1)'));
```

Comment:
Several instances of spaces instead of a tab in the line-start.

Reply:
fixed